dev/second_assignment/src/main/java/com/pironeer/week2_1/controller/CommentController.java

```diff
@@ -0,0 +1,63 @@
+package com.pironeer.week2_1.controller;
+
+import com.pironeer.week2_1.dto.request.CommentCreateRequest;
+import com.pironeer.week2_1.dto.request.CommentUpdateRequest;
+import com.pironeer.week2_1.dto.response.CommentResponse;
+import com.pironeer.week2_1.service.CommentService;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/comments")
+@RequiredArgsConstructor
+@Tag(name = "Comment", description = "댓글 API")
+public class CommentController {
+
+    private final CommentService commentService;
+
+    @Operation(summary = "댓글 생성", description = "새로운 댓글을 생성합니다.")
+    @PostMapping
+    public ResponseEntity<Void> createComment(@RequestBody CommentCreateRequest request) {
```

Comment:
```suggestion
    public ResponseEntity<Void> createComment(@Valid @RequestBody CommentCreateRequest request) {
```
`@Valid`와 의존성을 추가해 주셔야, CommentCreateRequest에 대한 유효성 검증이 가능합니다 !